Arrowgene.Ddon.Shared/Model/Equipment.cs

```diff
@@ -8,7 +8,7 @@ namespace Arrowgene.Ddon.Shared.Model
     #nullable enable
     public class Equipment
     {
-        private static readonly byte EQUIP_SLOT_NUMBER = 15;
+        public static readonly byte EQUIP_SLOT_NUMBER = 15;
```

Comment:
What does this constant represent exactly? I feel this name is not descriptive to what it is.

Reply:
It's the total number of equipment slots: weapons armor, and jewelry. What would be a better name for it?

Reply:
I see. I saw you adding it to some things like
```csharp
storageSlot += Equipment.EQUIP_SLOT_NUMBER;
```

Maybe it should be called `MAX_NUM_EQUIP_SLOTS`, `TOTAL_EQUIP_SLOTS` or something similar? I was reading `EQUIP_SLOT_NUMBER` initially as the slot which the item would be equipped to.